### PR TITLE
[New sample] Display OGC API collection

### PR DESCRIPTION
**Pull request by shubham7109**

### CODE REVIEW

**Review by TADraeseke — CHANGES_REQUESTED** [review]
Thanks @shubham7109 - This is a clean and very readable sample! Nice work!

Just a few small bits commented in line.

File: kotlin/display-ogc-api-collection/src/main/java/com/esri/arcgisruntime/sample/displayogcapicollection/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                val currentExtent = mapView.visibleArea.extent
+
+                // create a query based on the current visible extent
+                val visibleExtentQuery = QueryParameters()
+                visibleExtentQuery.geometry = currentExtent
+                visibleExtentQuery.spatialRelationship =
+                    QueryParameters.SpatialRelationship.INTERSECTS
+                // set a limit of 5000 on the number of returned features per request, the default on some services
+                // could be as low as 10
+                visibleExtentQuery.maxFeatures = 5000
```
Comment (TADraeseke) [inline kotlin/display-ogc-api-collection/src/main/java/com/esri/arcgisruntime/sample/displayogcapicollection/MainActivity.kt:120]:
```suggestion
                val visibleExtentQuery = QueryParameters().apply {
               geometry = currentExtent
                spatialRelationship =
                    QueryParameters.SpatialRelationship.INTERSECTS
                // set a limit of 5000 on the number of returned features per request, the default on some services
                // could be as low as 10
                maxFeatures = 5000
                }
```

Just spotted a good opportunity to use a scoping function here :)

**Comment by shubham7109** [comment]
- Resolved PR suggestions
- Added SC image
- Added `README.metadata.json` file
- Need input for `provision_from` & `provision_to` fields.

File: kotlin/display-ogc-api-collection/README.metadata.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+    "category": "TODO",
```
Comment (TADraeseke) [inline kotlin/display-ogc-api-collection/README.metadata.json:2]:
```suggestion
    "category": "Layers",
```

File: kotlin/display-ogc-api-collection/README.metadata.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        "feature table",
+        "service",
+        "table",
+        "web",
+        "OgcFeatureCollectionTable",
+        "QueryParameters"
+    ],
+    "language": "kotlin",
+    "provision_from": "TODO",
+    "provision_to": "TODO",
```
Comment (TADraeseke) [inline kotlin/display-ogc-api-collection/README.metadata.json:23]:
```suggestion
```

File: kotlin/display-ogc-api-collection/build.gradle [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    buildFeatures {
+        viewBinding true
+    }
+
```
Comment (TADraeseke) [inline kotlin/display-ogc-api-collection/build.gradle:25]:
```suggestion
```

File: kotlin/display-ogc-api-collection/src/main/java/com/esri/arcgisruntime/sample/displayogcapicollection/MainActivity.kt [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+import com.esri.arcgisruntime.mapping.view.MapView
+import com.esri.arcgisruntime.sample.displayogcapicollection.databinding.ActivityMainBinding
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol
+import com.esri.arcgisruntime.symbology.SimpleRenderer
+
+class MainActivity : AppCompatActivity() {
+
+    // define strings for the service URL and collection id
+    // note that the service defines the collection id which can be accessed
+    // via OgcFeatureCollectionInfo.getCollectionId().
```
Comment (TADraeseke) [inline kotlin/display-ogc-api-collection/src/main/java/com/esri/arcgisruntime/sample/displayogcapicollection/MainActivity.kt:26]:
```suggestion
    // via OgcFeatureCollectionInfo.getCollectionId()
```

**Review by TADraeseke — APPROVED** [review]
Just a few small things, then I think you're good to merge. Thanks @shubham7109!

**Pull request merged**